CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - Optional `chem` dependencies are lazily imported
 - Order of README sections
+- Move doc building utilities to separate file
```

Comment:
not sure if a changelog entry is really required for this PR but feel free to merge

Reply:
Well, it is a change, hence I included this. But I do not have any hard feelings about this, if you'd say "this is to minor", I can also remove this.

Reply:
Yes, but a change more like "rewrote a docstring" instead of "changed that function". Though not 100% clear in all cases, for my taste we should draw the line such that only changes are included that either 1) would be important for the user or 2) mark some architecture change that is important for us / the logic of the repo. The latter would be e.g. change from markdown to html documentation. 

But at the same time, as a general rule, having rather too much in the changelog is better than missing important parts. Just need a good compromise to avoid cluttering it...

Reply:
Agree with your two points for how to decide whether or not to put something in the CHANGELOG. So I'd remove this again from the CHANGELOG once I have the blessing of @Scienfitz

Reply:
*puts blessing*

Reply:
I have removed it. Will thus merge this once the tests are done.